dcrjson/dcrwalletextcmds.go

```diff
@@ -294,6 +294,47 @@ func NewPurchaseTicketCmd(fromAccount string, spendLimit float64, minConf *int,
 	}
 }
 
+// StartAutoBuyerCmd is a type handling custom marshaling and
+// unmarshaling of startautobuyer JSON RPC commands.
+type StartAutoBuyerCmd struct {
+	Passphrase        []byte
```

Comment:
I realize that other struct in this package don't specify lowercase json names via struct tags, but that is a mistake.  Take a look at all of the other files to see why I mean.

These should be `Passphrase       []byte  `json:"account"`